exchanges/bitfinex/bitfinex.go

```diff
@@ -87,6 +88,7 @@ type Bitfinex struct {
 	exchange.Base
 	WebsocketConn         *websocket.Conn
 	WebsocketSubdChannels map[int]WebsocketChanInfo
+	mu                    sync.Mutex
```

Comment:
Since this mutex is specifically related only to websocket send requests, should be named accordingly.